book/src/drivers.md

```diff
@@ -62,6 +62,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [SmartX](http://www.smartx.com/?locale=en) | `csi-smtx-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for SmartX ZBS Storage  | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
 [SandStone](https://github.com/sandstone-storage/sandstone-csi-driver) | `csi-sandstone-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for SandStone USP | Persistent | Read/Write Single Pod | Yes | 
 [StorageOS](https://docs.storageos.com/docs/platforms/kubernetes/install/) | `storageos` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [StorageOS](https://storageos.com/) | Persistent | Read/Write Multiple Pods | Yes |
+[TopoLVM](https://github.com/cybozu-go/topolvm)| `topolvm.cybozu.com` | v1.1 | A capacity-aware Container Storage Interface (CSI) Driver for LVM | Persistent | Read/Write Single Pod | Yes | Raw Block, Topology Aware, Capacity Aware, Volume Metrics
```

Comment:
This column is intended for common CSI driver features.  "Capacity Aware" and "Volume Metrics" are features supported by this driver but they are not CSI driver features so they should not be added to this column.  People who are interested in this driver can find out more about it from its website.

Reply:
Thank you for your comment. FIxed.